pkg/ocm/versions/versions.go

```diff
@@ -51,9 +56,40 @@ func GetVersions(client *cmv1.Client, channelGroup string) (versions []*cmv1.Ver
 		}
 		page++
 	}
+
+	// Sort list in descending order, ensuring the 'default' version at the top
+	sort.Slice(versions, func(i, j int) bool {
```

Comment:
what is the purpose of this? don't we already have the order by in the query?

Reply:
The query uses database sort, which doesn't understand semver, so it ends up sorting like
```
4.7.2
4.7.11
4.7.1
```